addon/models/osf-model.js

```diff
@@ -17,6 +17,7 @@ import HasManyQuery from 'ember-data-has-many-query';
 export default DS.Model.extend(HasManyQuery.ModelMixin, {
     links: DS.attr('links'),
     embeds: DS.attr('embed'),
+    errors: DS.attr(),
```

Comment:
That is helpful, thank you.  Another problem I have is that I need to know which error it received when it failed.  I am supposed to make project selector match OSF dashboard.

Dashboard tries to fetch parent. If it doesn't exist, then it doesn't show up in tree.  If it doesn't exist, but 403's, then it shows up as "private".  For example, Grandparent / Private/ Child.  The parent was private and it didn't show up. But to be able to distinguish, I need to know if it 404's or 403's.